tests/lib/snaps/test-snapd-timer-service/meta/snap.yaml

```diff
@@ -0,0 +1,11 @@
+name: test-snapd-timer-service
+version: 1.0
+apps:
+  regular-timer:
+    command: bin/loop regular-timer
+    daemon: simple
+    timer: 0:00-24:00/96
```

Comment:
To help casual reviewers. Is it correct to say that this means `# Run every 15 minutes.`?